mne/decoding/time_gen.py

```diff
@@ -249,9 +290,25 @@ def predict(self, epochs):
         # Store all testing times parameters
         self.test_times_ = test_times
 
-        # Prepare parallel predictions across time points
+        n_orig_epochs, _, n_times = X.shape
```

Comment:
why can't you call it `n_epochs` and conform to the convention in the rest of the code?